include/z64save.h

```diff
@@ -289,6 +289,9 @@ typedef struct {
     /* 0x1420 */ s16 worldMapArea;
     /* 0x1422 */ s16 sunsSongState; // controls the effects of suns song
     /* 0x1424 */ s16 healthAccumulator;
+#ifdef ENABLE_WIDESCREEN
+                 u8 isUsingWidescreen; // TODO: doesn't get saved?
```

Comment:
Saving is a bit of a mess, and not really shiftable yet due to substructs that are used as part of saving:
https://github.com/HackerN64/HackerOoT/blob/af4a250529f363467bc116b81322077b039c4c71/src/code/z_sram.c#L5-L73

Because only the struct `Save` is actually loaded when loading a file:
https://github.com/HackerN64/HackerOoT/blob/af4a250529f363467bc116b81322077b039c4c71/src/code/z_sram.c#L426

There is a PR somewhere in OoT decomp addressing this